include/RED4ext/Scripting/Natives/Generated/anim/Rig.hpp

```diff
@@ -4,7 +4,7 @@
 
 // This file is generated from the Game's Reflection data
 
-#include <RED4ext/Scripting/Natives/Rig.hpp>
+#include <RED4ext/Scripting/Natives/AnimRig.hpp>
```

Comment:
The path is incorrect, but it was fixed in #126. Just pull `include/RED4ext/Scripting/Natives/Generated/anim/Rig.hpp` and `include/RED4ext/Scripting/Natives/animRig.hpp` from the main branch and it should be good.